docs/en/compute/AWS-EMR.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: global
+title: Running Alluxio on EMR
+nickname: AWS EMR
+group: Data Applications
+priority: 0
+---
+
+This guide describes how to configure Alluxio to run on [AWS EMR](https://aws.amazon.com/emr/).
+
+* Table of Contents
+{:toc}
+
+## Overview
+
+AWS EMR provides great options for running clusters on-demand to handle compute workloads. It manages the
+deployment of various Hadoop Services and allows for hooks into these services for customizations. Alluxio
+can run on EMR to provide functionality above what EMRFS currently provides. Aside from the added performance
+benefits of caching, Alluxio also enables users to run compute workloads against on-premise storage or even a
+different cloud provider's storage i.e. GCS, Azure Blob Store. 
+
+## Prerequisites
+
+* Account with AWS
+* IAM Account with the default EMR Roles
+* Key Pair for EC2
+* An S3 Bucket
+* AWS CLI
+
+The majority of the pre-requisites can be found by going through the
+[AWS EMR Getting Started](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-gs.html) guide. An S3 bucket
+is needed as Alluxio's Root Under File System. If required, this can be reconfigured to be HDFS if needed.
+
+## Basic Setup
+
+To begin with, download the alluxio-emr.sh and alluxio-presto.json files from Github. These files will serve as the
+main mechanisms to change the Alluxio configuration in the future. Make sure that the AWS CLI is also set up and ready
+with the required AWS Access/Secret key.
+
+1. Edit the alluxio-emr.sh file and set the value for `alluxio.underfs.address=s3a://my-bucket/emr/alluxio`.
+2. Configure the below command with the required parameters
+
+```bash
+aws emr create-cluster --release-label emr-5.23.0 --instance-count 3 --instance-type <instance-type> --applications Name=Presto Name=Hive --name '<ClusterName>' --bootstrap-actions Path=s3://bucket/path/to//alluxio-emr.sh --configurations file:///path/to/file/alluxio-presto.json --ec2-attributes KeyName=<ec2-keypair-name>
```

Comment:
for `--bootstrap-actions`, does the Path need to be `s3://bucket/path/to//alluxio-emr.sh`? can we use http?

Reply:
at the moment bootstrap-actions can only be specified as an S3 URI